Dropdown: Add cursor pointer to DropdownStyled

ChengYanJin opened the pull request:
**Component**:
Dropdown, Button

**Description**:
When hover to the dropdown/button it's better to show the pointer so that the user can understand it's clickable.

**Design**:

ChengYanJin committed:
Dropdown: Add cursor pointer to DropdownStyled

When hover to the dropdown it's better to show the pointer so that the user can understand it's clickable.

bert-e commented:
# Hello chengyanjin,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/core-ui/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @carlito-scality

* @chengyanjin

* @rachedbenmustapha

* @nicolas2bert

* @monpote

ChengYanJin committed:
Button: Add the cursor pointer for available button

It's better to have the pointer when hover so that the user will understand it's clickable.

ChengYanJin committed:
update the test

carlito-scality commented on src/lib/components/button/Button.component.js:
```diff
@@ -97,7 +98,8 @@ export const ButtonStyled = styled.button`
         background-color: ${brandingTheme.background};
         color: ${brandingTheme[props.variant]};
-        
+        cursor:pointer;
```
IMO, we do not need it here.

ChengYanJin replied:
You are right, thanks!

carlito-scality commented on src/lib/components/button/Button.component.js:
```diff
@@ -179,7 +181,8 @@ ${props => {
         background-color: transparent;
         color: ${props.disabled ? brandLight : brandingTheme[props.variant]};
-        
+        cursor:pointer;
```
IMO, we do not need it here

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @carlito-scality

* @chengyanjin

* @rachedbenmustapha

* @nicolas2bert

* @monpote





The following reviewers are expecting changes from the author, or must review again:

* @carlito-scality

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @carlito-scality

* @chengyanjin

* @rachedbenmustapha

* @nicolas2bert

* @monpote

carlito-scality commented:
/approve

bert-e commented:
# Not Author



I'm afraid I cannot do that, @carlito-scality:

Only the author of the pull request @chengyanjin can use this command.

Please **delete** the corresponding comment so I can move on.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @carlito-scality

* @chengyanjin

* @rachedbenmustapha

* @nicolas2bert

* @monpote

ChengYanJin commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/1.0`




There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

carlito-scality commented:
/wait

carlito-scality committed:
update tests snapshots after rebasing development/1.0

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/1.0`




Please check the status of the associated issue None.

Goodbye chengyanjin.
